packages/features/eventtypes/components/tabs/recurring/RecurringEventController.tsx

```diff
@@ -24,11 +24,11 @@ export default function RecurringEventController({
   const [recurringEventState, setRecurringEventState] = useState<RecurringEvent | null>(
     formMethods.getValues("recurringEvent")
   );
-  /* Just yearly-0, monthly-1 and weekly-2 */
+
   const recurringEventFreqOptions = Object.entries(Frequency)
     .filter(([key, value]) => isNaN(Number(key)) && Number(value) < 3)
     .map(([key, value]) => ({
-      label: t(`${key.toString().toLowerCase()}`, { count: recurringEventState?.interval }),
+      label: t(`${key.toString().toLowerCase()}_one`),
```

Comment:
could you give some context on this change ?

Reply:
is there any impact on web ?

Reply:
we can use the hook useIsPlatform if needed to have a difference between the two

Reply:
> is there any impact on web ?
tested, web works fine

Reply:
rolled back